android/build.gradle

```diff
@@ -34,5 +34,6 @@ repositories {
 
 dependencies {
     compile 'com.facebook.react:react-native:+'
-    compile 'io.intercom.android:intercom-sdk-base:3.+'
+    compile 'io.intercom.android:intercom-sdk-base:4.1.4'
```

Comment:
Would using `4.+` be a good idea here?

Reply:
I used that fixed version as Intercom support suggested. Anyway, we shouldn't have any problem using `4.+` so go for it 👍